src/store/ui.ts

```diff
@@ -69,7 +69,9 @@ export const uiStore: VoiceVoxStoreOptions<UiGetters, UiActions, UiMutations> =
         state.uiLockCount++;
       },
       UNLOCK_UI(state) {
-        state.uiLockCount--;
+        if (state.uiLockCount !== 0) {
+          state.uiLockCount--;
+        }
```

Comment:
uiLockCountが0なときにUNLICK_UIが呼ばれるのは、おそらく意図していない挙動だと思います。
このときに`console.warn()`を出して、開発者は気づけるようにするというのはどうでしょう👀

ちょっとlinterが通るのかわかりませんが、こんな感じを意図しています
```suggestion
        if (state.uiLockCount !== 0) {
          state.uiLockCount--;
        }
        else {
          console.warn("UNLOCK_UI is called when state.uiLockCount == 0")
        }
```

Reply:
修正しました。
以下の様な感じでコンソールに表示されるようになります。
![Screenshot from 2022-04-04 11-32-05](https://user-images.githubusercontent.com/4476489/161464615-0e663bb3-6f85-4967-9fe2-3d52198d0843.png)

Reply:
```// eslint-disable-next-line no-console```を追加してwarning抑制しました．